### PR TITLE
Alerting docs: adds top-level landing page tiles

### PR DESCRIPTION
**Pull request by brendamuir**
built a landing page for the alerting product.

### CODE REVIEW

File: docs/sources/alerting/_index.md [hunk 1]
```diff
@@ -13,87 +13,54 @@ labels:
...
+      description: Choose how, when, and where to send your alert notifications.
+      height: 24
+    - title: Detect and respond
+      href: /docs/grafana/next/alerting/manage-notifications/
+      description: Monitor, respond to, and triage issues within your services.
+      height: 24
+    - title: Monitor
+      href: /docs/grafana/next/alerting/monitor/
+      description: Monitor your alerting metrics to ensure you identify potential issues before they become critical.
+      height: 24
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:49]:
```suggestion
hero:
  title: Grafana Alerting
  level: 1
  image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg
  width: 100
  height: 100
  description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
cards:
  title_class: pt-0 lh-1
  items:
    - title: Introduction
      href: ./fundamentals/
      description: Learn more about the fundamentals and available features that help you create, manage, and respond to alerts; and improve your team’s ability to resolve issues quickly.
      height: 24
    - title: Set up
      href: ./set-up/
      description: Set up your implementation of Grafana Alerting.
      height: 24
    - title: Configure alert rules
      href: ./alerting-rules/
      description: Create, manage, view, and adjust alert rules to alert on your metrics data or log entries from multiple data sources — no matter where your data is stored.
      height: 24
    - title: Configure notifications
      href: ./configure-notifications/
      description: Choose how, when, and where to send your alert notifications.
      height: 24
    - title: Detect and respond
      href: ./manage-notifications/
      description: Monitor, respond to, and triage issues within your services.
      height: 24
    - title: Monitor
      href: ./monitor/
      description: Monitor your alerting metrics to ensure you identify potential issues before they become critical.
      height: 24

```

File: docs/sources/alerting/_index.md [hunk 2]
```diff
@@ -13,87 +13,54 @@ labels:
 menuTitle: Alerting
 title: Grafana Alerting
 weight: 114
+hero:
+  title: Grafana Alerting
+  level: 1
+  image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg
+  width: 110
+  height: 110
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:21]: Results in better spacing before the horizontal rule.
```suggestion
  width: 100
  height: 100
```

File: docs/sources/alerting/_index.md [hunk 3]
```diff
@@ -13,87 +13,54 @@ labels:
...
+  description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
+cards:
+  title_class: pt-0 lh-1
+  items:
+    - title: Introduction
+      href: /docs/grafana-cloud/alerting-and-irm/alerting/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:27]: The previous link linked to itself in Cloud and to the Cloud landing page from OSS. I think this is meant to go to the page with the first heading "Introduction". The relative path below works in both Grafana OSS and Cloud.
```suggestion
      href: ./fundamentals/
```

File: docs/sources/alerting/_index.md [hunk 4]
```diff
@@ -13,87 +13,54 @@ labels:
...
+      description: Learn more about the fundamentals and available features that help you create, manage, and respond to alerts; and improve your team’s ability to resolve issues quickly.
+      height: 24
+    - title: Set up
+      href: /docs/grafana/next/alerting/set-up/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:31]: To work in both OSS and Cloud docs
```suggestion
      href: ./set-up/
```

File: docs/sources/alerting/_index.md [hunk 5]
```diff
@@ -13,87 +13,54 @@ labels:
...
+      description: Set up your implementation of Grafana Alerting.
+      height: 24
+    - title: Configure alert rules
+      href: /docs/grafana/next/alerting/alerting-rules/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:35]: To work in both OSS and Cloud docs
```suggestion
      href: ./alerting-rules/
```

File: docs/sources/alerting/_index.md [hunk 6]
```diff
@@ -13,87 +13,54 @@ labels:
...
+      description: Create, manage, view, and adjust alert rules to alert on your metrics data or log entries from multiple data sources — no matter where your data is stored.
+      height: 24
+    - title: Configure notifications
+      href: /docs/grafana/next/alerting/configure-notifications/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:39]: To work in both OSS and Cloud docs
```suggestion
      href: ./configure-notifications/
```

File: docs/sources/alerting/_index.md [hunk 7]
```diff
@@ -13,87 +13,54 @@ labels:
...
+      description: Choose how, when, and where to send your alert notifications.
+      height: 24
+    - title: Detect and respond
+      href: /docs/grafana/next/alerting/manage-notifications/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:43]: To work in both OSS and Cloud docs
```suggestion
      href: ./manage-notifications/
```

File: docs/sources/alerting/_index.md [hunk 8]
```diff
@@ -13,87 +13,54 @@ labels:
...
+      description: Monitor, respond to, and triage issues within your services.
+      height: 24
+    - title: Monitor
+      href: /docs/grafana/next/alerting/monitor/
```
Comment (jdbaldry) [inline docs/sources/alerting/_index.md:47]: To work in both OSS and Cloud docs
```suggestion
      href: ./monitor/
```

File: docs/sources/alerting/_index.md (hunk 3 above)
Comment (brendamuir) [reply docs/sources/alerting/_index.md:27]: hmm so i need all the links to work from both oss and cloud. should they all be changed?

File: docs/sources/alerting/_index.md (hunk 5 above)
Comment (brendamuir) [reply docs/sources/alerting/_index.md:35]: ok..there's my answer haha

**Review by jdbaldry — APPROVED** [review]
LGTM!

**Pull request merged**